### PR TITLE
Kevin db

### PR DESCRIPTION
**Pull request by Kevin-Joseph**

### CODE REVIEW

File: BCC/src/main/java/BNU/presentation/MainView.java [hunk 1]
```diff
@@ -101,6 +106,33 @@ public static void BuildLoginView(JFrame mainFrame, MainController controller) {
 		
 		//build center panel
 		
+		// auto close connection
+        try (Connection conn = DriverManager.getConnection(
+                "jdbc:postgresql://127.0.0.1:5432/BCC", "postgres", "aspartame")) {
```
Comment (MarkFuller1) [inline BCC/src/main/java/BNU/presentation/MainView.java]: All of this code is going to be replaced by the DB wrapper, so we can remove it from here

**Review by MarkFuller1 — CHANGES_REQUESTED** [review]
can you remove the lines

**Comment by Kevin-Joseph** [comment]
Yeah I can remove these lines, I will figure out how to.

**Review by Kevin-Joseph — COMMENTED** [review]
Took out the db connection code.

**Review by MarkFuller1 — APPROVED** [review]

**Pull request merged**